### ISSUE TITLE
Accessibility - Enforce manual page zoom does not work/survive app restart

### ISSUE THREAD

**Issue by smileatom** [duckduckgo/iOS#1137]
Hi, 

I cant use mobile browsers that dont allow force zoom override since many websites lock down very small fonts and I cant read them.

Sadly, the implementation of Accessibility/Enforce manual page zoom does not work.  If you enable it while a webpage is open it will mostly work if you are lucky.  However it will not work after a restart of the DDG app even though the setting is still enabled in settings. 

This affects both iOS and android.  You should test stuff.

I will use chrome until this bug is fixed. 

Apologies for being a spaz, I believe this is only on Android.  iOS/Safari webview does allow pinch zoom on sites that restrict it.

I will file this on the ddg android repo
A good test site is https://techmeme.com

**Comment by samsymons** [comment]
Hey @smileatom,

Thanks for writing in – since you mentioned this is specifically an Android issue, I'm closing this out. Please do let us know if you run into any other issues though!

Thanks!
